<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerMain</class>
 <widget class="QMainWindow" name="ViewerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1395</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="cgQSeismicPlotWidget" name="seisdata" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1395</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Movie"/>
    <addaction name="separator"/>
    <addaction name="action_Duplicate"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionQuickPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="action_Seismic_data_setting"/>
    <addaction name="action_Plot_setting"/>
    <addaction name="action_Color_map_setting"/>
    <addaction name="action_Trace_details"/>
    <addaction name="actionConfig_axis"/>
   </widget>
   <widget class="QMenu" name="menu_Movie">
    <property name="title">
     <string>&amp;Movie</string>
    </property>
    <addaction name="action_Forward"/>
    <addaction name="action_Backward"/>
    <addaction name="action_Stop"/>
    <addaction name="actionForward_1"/>
    <addaction name="action_1"/>
    <addaction name="actionTo_Beginning"/>
    <addaction name="actionTo_End"/>
    <addaction name="action_Select_frame"/>
    <addaction name="actionSet_s_peed"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_About"/>
    <addaction name="separator"/>
    <addaction name="action_Shortcuts"/>
    <addaction name="action_Known_bugs"/>
   </widget>
   <widget class="QMenu" name="menu_Quick_configure">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_Zoom_out"/>
    <addaction name="action_Mode_switch"/>
    <addaction name="action_Vertical_flip"/>
    <addaction name="action_Horizontal_flip"/>
    <addaction name="action_Parity_flip"/>
    <addaction name="action_Horizontal_grid"/>
    <addaction name="action_Color_map"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Quick_configure"/>
   <addaction name="menu_Movie"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainBar">
   <property name="windowTitle">
    <string>Short cut actions</string>
   </property>
   <property name="toolTip">
    <string>Toggle wiggle, density, or both</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionOpen_Movie"/>
   <addaction name="actionQuickPrint"/>
   <addaction name="action_Duplicate"/>
   <addaction name="separator"/>
   <addaction name="action_Seismic_data_setting"/>
   <addaction name="action_Plot_setting"/>
   <addaction name="action_Trace_details"/>
   <addaction name="actionConfig_axis"/>
   <addaction name="separator"/>
   <addaction name="action_Color_map_setting"/>
   <addaction name="action_Mode_switch"/>
   <addaction name="action_Color_map"/>
   <addaction name="action_Parity_flip"/>
   <addaction name="action_Vertical_flip"/>
   <addaction name="action_Horizontal_flip"/>
   <addaction name="action_Zoom_out"/>
   <addaction name="action_Horizontal_grid"/>
  </widget>
  <widget class="QToolBar" name="movieBar">
   <property name="windowTitle">
    <string>Movie control</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTo_Beginning"/>
   <addaction name="action_Backward"/>
   <addaction name="action_Stop"/>
   <addaction name="action_Forward"/>
   <addaction name="actionTo_End"/>
   <addaction name="separator"/>
   <addaction name="action_1"/>
   <addaction name="actionForward_1"/>
   <addaction name="separator"/>
   <addaction name="action_Select_frame"/>
   <addaction name="actionSet_s_peed"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/open_file.png</normaloff>:/img/img/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a data file</string>
   </property>
  </action>
  <action name="actionOpen_Movie">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/open_movie.png</normaloff>:/img/img/open_movie.png</iconset>
   </property>
   <property name="text">
    <string>Open Movie</string>
   </property>
   <property name="toolTip">
    <string>Open a directory containing movie data files</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Seismic_data_setting">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/data_properties_map.png</normaloff>:/img/img/data_properties_map.png</iconset>
   </property>
   <property name="text">
    <string>Data setting</string>
   </property>
   <property name="toolTip">
    <string>Data setting for the plot</string>
   </property>
  </action>
  <action name="action_Plot_setting">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/plot_settings.png</normaloff>:/img/img/plot_settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Plot setting</string>
   </property>
   <property name="toolTip">
    <string>Plot parameterization setting</string>
   </property>
  </action>
  <action name="action_Color_map_setting">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/color_map_editor.png</normaloff>:/img/img/color_map_editor.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Color map setting</string>
   </property>
  </action>
  <action name="action_Trace_details">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/data_detail.png</normaloff>:/img/img/data_detail.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Trace details</string>
   </property>
  </action>
  <action name="actionQuickPrint">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/print_whole.png</normaloff>:/img/img/print_whole.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quick print</string>
   </property>
  </action>
  <action name="action_Forward">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/forward.png</normaloff>:/img/img/forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Forward</string>
   </property>
  </action>
  <action name="action_Backward">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/backward.png</normaloff>:/img/img/backward.png</iconset>
   </property>
   <property name="text">
    <string>Backward</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/stop.png</normaloff>:/img/img/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionForward_1">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/one_forward.png</normaloff>:/img/img/one_forward.png</iconset>
   </property>
   <property name="text">
    <string>+ 1</string>
   </property>
   <property name="toolTip">
    <string>Forward one frame</string>
   </property>
  </action>
  <action name="action_1">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/one_backward.png</normaloff>:/img/img/one_backward.png</iconset>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
   <property name="iconText">
    <string>back one</string>
   </property>
   <property name="toolTip">
    <string>go back one frame</string>
   </property>
  </action>
  <action name="actionTo_Beginning">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/begin.png</normaloff>:/img/img/begin.png</iconset>
   </property>
   <property name="text">
    <string>To Beginning</string>
   </property>
  </action>
  <action name="actionTo_End">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/end.png</normaloff>:/img/img/end.png</iconset>
   </property>
   <property name="text">
    <string>To End</string>
   </property>
  </action>
  <action name="action_Zoom_out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/zoom.png</normaloff>:/img/img/zoom.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom out</string>
   </property>
  </action>
  <action name="action_Select_frame">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/change_position.png</normaloff>:/img/img/change_position.png</iconset>
   </property>
   <property name="text">
    <string>Select f&amp;rame</string>
   </property>
  </action>
  <action name="actionSet_s_peed">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/change_speed.png</normaloff>:/img/img/change_speed.png</iconset>
   </property>
   <property name="text">
    <string>Set s&amp;peed</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="action_Duplicate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/new_window.png</normaloff>:/img/img/new_window.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Duplicate</string>
   </property>
   <property name="toolTip">
    <string>Open new viewer with this data</string>
   </property>
  </action>
  <action name="actionConfig_axis">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/axis_definition.png</normaloff>:/img/img/axis_definition.png</iconset>
   </property>
   <property name="text">
    <string>config axis</string>
   </property>
  </action>
  <action name="action_Shortcuts">
   <property name="text">
    <string>&amp;Shortcuts</string>
   </property>
  </action>
  <action name="action_Mode_switch">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/color_line_toggle.png</normaloff>:/img/img/color_line_toggle.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Mode switch</string>
   </property>
  </action>
  <action name="action_Parity_flip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/polarity_toggle.png</normaloff>:/img/img/polarity_toggle.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Parity flip</string>
   </property>
  </action>
  <action name="action_Horizontal_grid">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/horizontal_grid.png</normaloff>:/img/img/horizontal_grid.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Horizontal grid</string>
   </property>
   <property name="toolTip">
    <string>Toggle horizontal grid</string>
   </property>
  </action>
  <action name="action_Color_map">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/color_map_toggle.png</normaloff>:/img/img/color_map_toggle.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Color map</string>
   </property>
   <property name="toolTip">
    <string>Toggle color map</string>
   </property>
  </action>
  <action name="action_Horizontal_flip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/flip_horizontal.png</normaloff>:/img/img/flip_horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Horizontal flip</string>
   </property>
  </action>
  <action name="action_Vertical_flip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/img/flip_vertical.png</normaloff>:/img/img/flip_vertical.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Vertical flip</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_Known_bugs">
   <property name="text">
    <string>Known bugs</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cgQSeismicPlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtSeismic/QSeismicPlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
